### DISCUSSION TITLE
TOP LANGs does count languages in repos I contributed to?

### DISCUSSION THREAD

**Discussion by assebc** [anuraghazra/github-readme-stats#1916] [Q&A] [answered]
Today I deployed my own version of github-readme-stats, first created by anuraghazra, after being able to count my private repos languages as I wanted to, Im asking myself if it also count repositories that I contributed to. If it does, how do I change that?

**Comment by rickstaa** [comment, marked as answer]
@assebc Thanks for your question. Including data from organizations is our most requested feature (see https://github.com/anuraghazra/github-readme-stats/issues/1). A PR is open for supporting this feature, but we have not yet merged it. It can, however, already be used on your own Vercel instance by following https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174.

**Reply by assebc** [reply]
@rickstaa Thanks for the answer, just answering to make sure about it because if it did I would have duplicated code conting to top-langs and certainly I do not want that!

**Reply by rickstaa** [reply]
@assebc not sure if I understand your question but #1122 counts languages per repository so you should not encounter any problems. You can check out the syntax of the PR using the [graphql explorer] (https://docs.github.com/en/graphql/overview/explorer).

**Reply by assebc** [reply]
@rickstaa I always save a private repository of certain project that I contributed to in casa later on I want to give it a little of my own effect, I already took a look at that PR's, thanks for the help!

**Reply by rickstaa** [reply]
Ah, I see, then it makes sense! Using forking for these copies should not cause problems since we currently do not include stats from forks. It will only be counted double if you create an orphan branch. In that case, you can use the `exclude_repo` option to filter out these orphan repositories (see https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options). 🤔 If your problem is solved, please consider accepting my answer so that it shows up in the overview as answered. 🤓
